tests/decorators/sigs.go

```diff
@@ -0,0 +1,29 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright the KubeVirt Authors.
+ *
+ */
+
+package decorators
+
+import "github.com/onsi/ginkgo/v2"
+
+func SIG(identifier string, decorator ginkgo.Labels, text string, args ...interface{}) (extendedText string, newArgs []interface{}) {
+	newArgs = []interface{}{decorator}
+	newArgs = append(newArgs, args...)
+	extendedText = identifier + " " + text
+	return
+}
```

Comment:
Can't this be (something like):
```suggestion
func SIG(identifier string, text string, args ...interface{}) (extendedText string, newArgs []interface{}) {
	newArgs = args
	extendedText = identifier + " " + text
	return
}
```

and then use it as:
```
func KWOK(text string, args ...interface{}) (extendedText string, newArgs []interface{}) {
	return decorators.SIG("[sig-performance]", text, Serial, decorators.SigPerformance, Label("KWOK"), args)
}
```
instead of:
```
func KWOK(text string, args ...interface{}) (extendedText string, newArgs []interface{}) {
	return decorators.SIG("[sig-performance]", decorators.SigPerformance, text, Serial, Label("KWOK"), args)
}
```
In this way, text construction arguments are at the beginning and decorators/labels are at the end (and an `append` execution) WDYT?

Reply:
Great suggestion, done! :100: